tests/test_full_example_duckdb.py

```diff
@@ -238,29 +288,13 @@ def test_small_example_duckdb(tmp_path):
         "retain_intermediate_calculation_columns": True,
     }
 
-    linker = DuckDBLinker(
-        df,
-        connection=os.path.join(tmp_path, "duckdb.db"),
-        output_schema="splink_in_duckdb",
-    )
-    linker.load_settings(settings_dict)
+    linker = DuckDBLinker(df, settings_dict)
 
     linker.estimate_u_using_random_sampling(max_pairs=1e6)
-    linker.estimate_probability_two_random_records_match(
-        ["l.email = r.email"], recall=0.3
-    )
     blocking_rule = "l.full_name = r.full_name"
     linker.estimate_parameters_using_expectation_maximisation(blocking_rule)
 
```

Comment:
Largely because a lot of this is duplication of tests. We are already testing this functionality in an integration test above.

This original test is from [here](https://github.com/moj-analytical-services/splink/pull/1156).

It looks to me as if it's just meant to be an integration test for `columns_reversed_level` and a lot of the surrounding test is superfluous.

I'm happy to undo this if you think it's beneficial to the test environment. I'd probably suggest if I do so, I just add `estimate_probability_two_random_records_match` back in and leave the rest out.

It doesn't add anything to the test suite.

Reply:
Ah yeah sorry, didn't realise this was mostly just duplicated above. Yep, this looks fine then 👍